### PR TITLE
Futz with the build

### PR DESCRIPTION
**Pull request by armanbilge**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -15,12 +15,6 @@ val Scala213 = "2.13.8"
 ThisBuild / crossScalaVersions := Seq(Scala213, "2.12.15", "3.0.2")
 ThisBuild / scalaVersion := Scala213 // the default Scala
 
-ThisBuild / homepage := Some(url("https://github.com/TimWSpence/cats-stm"))
-
-ThisBuild / scmInfo := Some(
-  ScmInfo(url("https://github.com/TimWSpence/cats-stm"), "git@github.com:TimWSpence/cats-stm.git")
-)
```
Comment (armanbilge) [inline build.sbt:22]: This is set automatically. Actually with more info then you've set here I believe 😆

File: build.sbt [hunk 2]
```diff
@@ -59,9 +54,6 @@ lazy val core = crossProject(JVMPlatform, JSPlatform)
     import cats.effect.implicits._
     import cats.effect.unsafe.implicits.global
     """)
-  .settings(
-    javacOptions ++= Seq("-source", "1.8", "-target", "1.8")
-  )
```
Comment (armanbilge) [inline build.sbt:64]: You don't have java code in your project do you?

Comment (armanbilge) [reply build.sbt:64]: Btw this will also be set automatically in a future release.

Comment (armanbilge) [reply build.sbt:64]: Woops, you do have Java.

File: build.sbt [hunk 3]
```diff
@@ -14,12 +14,7 @@ ThisBuild / tlSonatypeUseLegacyHost := false
 val Scala213 = "2.13.8"
 ThisBuild / crossScalaVersions := Seq(Scala213, "2.12.15", "3.0.2")
 ThisBuild / scalaVersion := Scala213 // the default Scala
-
-ThisBuild / homepage := Some(url("https://github.com/TimWSpence/cats-stm"))
-
-ThisBuild / scmInfo := Some(
-  ScmInfo(url("https://github.com/TimWSpence/cats-stm"), "git@github.com:TimWSpence/cats-stm.git")
-)
+ThisBuild / tlJdkRelease := Some(8)
```
Comment (armanbilge) [inline build.sbt:17]: There you go :)

Comment (TimWSpence) [reply build.sbt:17]: Does this set the javac target option?

Comment (armanbilge) [reply build.sbt:17]: Yes.

Comment (armanbilge) [reply build.sbt:17]: Well, actually no :) it sets the `--release` flag which has superseded that, but anyway.

File: build.sbt [hunk 4]
```diff
@@ -34,6 +29,7 @@ lazy val `cats-stm` = tlCrossRootProject
     core,
     benchmarks,
     docs,
+    unidoc,
```
[the remark this answers is not in the document]
Comment (TimWSpence) [reply build.sbt:32]: Haha yeah, it's on my TODO list. Thanks!

**Comment by TimWSpence** [comment]
Thanks so much @armanbilge! :)

**Pull request merged**